account_invoice_period_usability/README.md

```diff
@@ -0,0 +1,25 @@
+Display in the invoice supplier form the fiscal period next to the invoice date
```

Comment:
Can you add the licence badge at the beginning ?

.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License